auth/api/iam/metadata.go

```diff
@@ -34,3 +38,23 @@ func authorizationServerMetadata(identity url.URL) OAuthAuthorizationServerMetad
 		ClientIdSchemesSupported: clientIdSchemesSupported,
 	}
 }
+
+// clientMetadata should only be used for dids managed by the node. It assumes the provided identity URL is correct.
+func clientMetadata(identity url.URL) OAuthClientMetadata {
+	softwareID, softwareVersion, _ := strings.Cut(core.UserAgent(), "/")
+	return OAuthClientMetadata{
+		//RedirectURIs:            nil,
+		TokenEndpointAuthMethod: "client_secret_basic", // TODO: this is the default value, what do we support
+		GrantTypes:              grantTypesSupported,
+		ResponseTypes:           responseTypesSupported,
+		//Scope:                   "",
```

Comment:
remove comments

Reply:
I left them as comments to see what fields still need to added, but I will remove them.

Reply:
Left the remaining values commented. They either need to be set by the time these flows are implmented or removed.